src/test-jdk14/java/tools/jackson/databind/failing/RecordUpdate3079Test.java

```diff
@@ -7,7 +7,7 @@
 // 01-Dec-2022, tatu: Alas, fails on JDK 17
 public class RecordUpdate3079Test extends BaseMapTest
 {
-    record IdNameRecord(int id, String name) { }
+    public record IdNameRecord(int id, String name) { }
```

Comment:
Hmmh. While I understand this will allow passing the test, it just hides regression in functionality: with 2.x these would work without being public.

I think I'll try visibility checked change instead, to see if that can work.